services/components/select.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint max-nested-callbacks: [1, 5] */
```

Comment:
Yeah, but deleting the code exposed a linting issue. we massage the max-nested-callbacks a little bit in a few of our tests